docs/installation/setup/ubuntu.rst

```diff
@@ -20,13 +20,13 @@ GDAL, HDF5, and netCDF4::
 
     sudo apt-get install libgdal-dev libhdf5-serial-dev libnetcdf-dev
 
-Install the latest Postgres version `available <https://packages.ubuntu.com/search?keywords=postgresql>` for your
+Install the latest Postgres version `available <https://packages.ubuntu.com/search?keywords=postgresql>`_ for your
 Ubuntu distribution, eg::
 
-    sudo apt-get install postgresql-12
+    sudo apt-get install postgresql-14
 
     # Optionally, Postgis too
```

Comment:
Optionally, postgis too  (required for the postgis/experimental index driver).

Reply:
See the next lines